c2corg_ui/templates/helpers/view.html

```diff
@@ -81,45 +81,91 @@
 </%def>
 
 <%def name="show_associated_waypoints(document, type=None)">\
-  <ul>
   % if 'waypoints' in document['associations']:
     % for w in document['associations']['waypoints']:
       % if type is None or type == w['waypoint_type']:
-      <li class="card">
+      <div class="list-item">
         <a href="${request.route_url('waypoints_view', id=w['document_id'], lang=w['locales'][0]['lang'])}">${w['locales'][0]['title']}</a>
         <app-delete-association parent-id="${document['document_id']}" child-id="${w['document_id']}"></app-delete-association>
-      </li>
+      </div>
       % endif
     % endfor
   % endif
-  </ul>
 </%def>
 
 <%def name="show_associated_routes(document)">\
-  <ul>
   % if 'routes' in document['associations']:
     % for r in document['associations']['routes']:
-    <li class="card">
+    <div class="list-item">
       <a href="${request.route_url('routes_view', id=r['document_id'], lang=r['locales'][0]['lang'])}">${r['locales'][0]['title']}</a>
       <app-delete-association parent-id="${document['document_id']}" child-id="${r['document_id']}"></app-delete-association>
-    </li>
+    </div>
     % endfor
   % endif
-  </ul>
 </%def>
 
 <%def name="show_maps(document)">\
-  <ul>
-  % for m in document['maps']:
-    <li>${m['editor']} ${m['locales'][0]['title']}</li>
-  % endfor
-  </ul>
+  % if len(document['maps']) > 0:
+    <ul>
+    % for m in document['maps']:
+      <li>${m['editor']} ${m['locales'][0]['title']}</li>
+    % endfor
+    </ul>
+  % else :
+  <p translate>No maps associated</p>
+  % endif
 </%def>
 
 <%def name="show_areas(document)">\
-  <ul>
-  % for a in document['areas']:
-    <li>${a['area_type']} ${a['locales'][0]['title']}</li>
-  % endfor
-  </ul>
+  % if len(document['areas']) > 0:
+    <ul>
+    % for a in document['areas']:
+    <li><span translate>${a['area_type']}</span>: <b>${a['locales'][0]['title']}</b></li>
```

Comment:
Come on! Is this that easy to create an alias for a directive? That the solution I've been looking for (not that actively :P) for weeks!

That would be much more convenient then the current solution (``{{mainCtrl.translate('${a['area_type']}')}}``) which is verbose and adds ``{{}}`` until the translation is done (which is ugly).

I don't think ``translate-enum`` is a relevant name since we would need it for other "variable" content. What about ``translate-var``?